forc-pkg/src/pkg.rs

```diff
@@ -2039,7 +2065,7 @@ pub fn build_with_options(build_options: BuildOpts) -> Result<Built> {
     let outputs = member_filter.filter_outputs(&build_plan, outputs);
 
     // Build it!
-    let mut built_workspace = HashMap::new();
+    let mut built_workspace = Vec::new();
```

Comment:
Nice, the `Vec` is probably a better fit here as we get to retain order, deduplicate the pkg name, and keying by pkg name probably isn't useful anyways.